src/Components/DataRow.jsx

```diff
@@ -3,14 +3,19 @@ import PropTypes from 'prop-types';
 
 class DataRow extends Component {
 
+    handleOnClick(row) {
+        if (!!onClick) {
+            return onClick(row);
+        }
+    }
+
     render() {
-        const props = this.props;
-        const row = props.row;
+        const { row, fields } = this.props;
 
         return (
-            <tr>
+            <tr onClick={() => handleOnClick(row)}>
                 { this.renderCheckboxCell(row.id) }
-                { props.fields.map(field => this.renderCell(field, row)) }
+                { fields.map(field => this.renderCell(field, row)) }
```

Comment:
Don't use anonymous functions in the render method - they cause unnecessary re-renders and can negatively impact performance.

Reply:
This was already in the library, I just deconstructed the object